magellan-sample-migration/src/test/resources/robolectric.properties

```diff
@@ -1 +1,2 @@
-sdk=28
\ No newline at end of file
+sdk=28
+application=com.wealthfront.magellan.sample.migration.TestSampleApplication
```

Comment:
Robolectric will pick this up automatically since it's looking out for classes named "TestBlahApplication", but good to be explicit anyways